entrypoints/content/components/secondary/comment/CommentAction.tsx

```diff
@@ -0,0 +1,71 @@
+
+import { useState } from 'react'
+
+
+// Components::
+import { ArrowBigDown, ArrowBigUp, Ellipsis, Forward, MessageSquare } from 'lucide-react'
+
+// Types:
+import { VoteCount } from 'types/votes'
+
+interface VoteCountProps {
+  voteCount: VoteCount
+}
+
+const CommentAction: React.FC<VoteCountProps> = ({ voteCount: { up = 0, down = 0 } }) => {
+
+  const [voteCount, setVotecount] = useState(up - down)
+  const [isUpVoteActive, setUpVoteActive] = useState(true)
```

Comment:
`upVote` is not wrong, but the entire application is using the term `upvote` (considering it as a single word), so the same thing would also go here
same for downvote too

Reply:
fixed